package.json

```diff
@@ -374,14 +377,15 @@
     "**/kind-of": "^6.0.3",
     "**/glob-parent": "^5.1.2",
     "**/ansi-regex": "^5.0.1",
-    "**/yargs-parser": "13.1.2",
     "@babel/runtime": "^7.15.4",
-    "punycode": "1.4.1",
```

Comment:
Previously we had to downgrade punycode because newer versions of punycode caused errors in IE 11. Make sure to test thoroughly in IE 11